optuna/structs.py

```diff
@@ -16,6 +16,9 @@
     from optuna.distributions import BaseDistribution  # NOQA
 
 
+# The use of the structs.StudyDirectino is deprecated and
```

Comment:
Maybe you're aware of this already but the maximum line length is 99 chars, so if you want, you could wrap later.

Reply:
Thank you for your comment. I will change it as it has 99 line length.

Reply:
Great!